torch/csrc/jit/codegen/cuda/interface.h

```diff
@@ -25,6 +25,7 @@ struct CudaFuserInterface {
   void (*fn_run_n_s_)(const Node*, Stack&) = nullptr;
   void (*fn_fuse_graph_)(std::shared_ptr<Graph>&) = nullptr;
   bool (*fn_can_fuse_n_)(const Node*) = nullptr;
+  void (*fn_insert_profile_inodes_)(ProfilingRecord* pr) = nullptr;
```

Comment:
My interpretation of the google style is that the suffix rule only differs between `class` and `struct`, but it irrelevant to access level. https://google.github.io/styleguide/cppguide.html#Variable_Names
I am aware that other style guidelines do put suffix based on that. :shrug:

Reply:
`_` suffix is only for private data members (the Google style definition is a bit confusing since it assumes only public members on structs and only private on classes - but the suffix reflects the access level, not struct vs. class)